packages/manager/src/components/ActivityStream/ActivityStream.tsx

```diff
@@ -47,15 +48,25 @@ const activities = {
       <span style={{ fontWeight: '600' }}>{propertyIdentifier}</span> and spa :{' '}
       <span style={{ fontWeight: '600' }}>{props.applicationIdentifier}</span> for{' '}
       <span style={{ fontWeight: '600' }}>{props.env} </span> environment within{' '}
-      {message.split(' ')[3]}s{' '}
+      {message.split(' ')[3]}s
```

Comment:
Something like this might help
```suggestion
      {message?.split(' ')[3]}s
```

Reply:
Fixed